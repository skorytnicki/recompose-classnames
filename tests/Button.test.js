import React from "react";
import {Button} from "./Button";
import renderer from "react-test-renderer";

test("Button has class isPrimary", () => {
    const component = renderer.create(
        <Button isPrimary>Jug Button</Button>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});