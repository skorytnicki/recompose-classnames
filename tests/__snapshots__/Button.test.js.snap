// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button has class 1`] = `
<button
  className="isPrimary"
  isPrimary={true}
>
  Jug Button
</button>
`;

exports[`Button has class isPrimary 1`] = `
<button
  className="isPrimary"
  isPrimary={true}
>
  Jug Button
</button>
`;
